MLS.Agent.Tests/TemplateTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using WorkspaceServer;
+using Xunit;
+using WorkspaceServer.Tests;
+using System.IO;
+using FluentAssertions;
+using System.Threading.Tasks;
+using System.CommandLine;
+using MLS.Agent.CommandLine;
+using MLS.Agent.Tools;
+using Microsoft.DotNet.Try.Protocol.Tests;
+using System.Linq;
+
+namespace MLS.Agent.Tests
```

Comment:
These tests should probably go into a separate project specific to the templates.